apex-benchmarks/src/main/java/apex/benchmark/DimensionTuple.java

```diff
@@ -18,10 +18,15 @@
   public String campaignId;
 
   public long eventTime;
-  public long clicks;
+  public long clicks = 1;
+  
+  protected static boolean autoGeneratorCampaignId = true;
 
   public static DimensionTuple fromTuple(Tuple tuple)
   {
+    if(tuple.campaignId == null && autoGeneratorCampaignId) {
+      tuple.campaignId = "1111111111111111111";
```

Comment:
The campaignId should not empty in normal case. 
It used for unit test where campainId will not populated. choose a invalid campainId for easier identify if any problem